lib/Zim.js

```diff
@@ -99,6 +100,24 @@ Zim.prototype.computeZimName = function() {
     return (this.publisher ? this.publisher.toLowerCase() + '.' : '') + this.env.computeFilenameRadical(false, true, true);
 };
 
+Zim.prototype.computeZimTags = function() {
+    var tags = [];
+
+    /* Mediawiki hostname radical */
+    var mwUrlHostParts = urlParser.parse(this.env.mw.base).host.split(".");
+    var mwUrlHostPartsTag = mwUrlHostParts.length > 1 ?
+	mwUrlHostParts[mwUrlHostParts.length - 2] : mwUrlHostParts[mwUrlHostParts.length - 1]
```

Comment:
I don't have an opinion on this .. depends on what the requirement are for tags.

Reply:
Kind of automatic solution to "categorize" somehow ZIM files.